nuclear/message/generate_python_messages.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+import os
+import pkgutil
+import shutil
+import sys
+
+import google.protobuf.message
+
+# TODO(Kip): This file is broken. See `messages.txt` in the build directory. It's full of duplicates. It's created
+# inside this file. We need to fix the custom command which calls this, and we need to fix this.
```

Comment:
It's not broken. `meesages.txt` is just a log file, it looks the way it is meant to look